README.md

```diff
@@ -141,7 +141,7 @@ quark -a first.apk -a second.apk -C
 <img src="https://i.imgur.com/ClRWOei.png"/>
 
 ### Parallelizing Quark
-Now Quark supports multiprocessing for analyzing APKs parallelly, by adding the option `--multi-process` you can set the number of processes. This value won't exceed the number of CPUs - 1 (and will default to it if any larger value is provided) . If no value is provided the, 1 will be used as a default value.
+Now Quark supports multiprocessing for analyzing APKs parallelly, by adding the option `--multi-process` you can set the number of processes. The Quark engine automatically limits this value to be less than or equal to the number of CPUs - 1. This restriction is done to avoid the CPU from running out memory. If no value is provided, 1 will be used as a default value.
```

Comment:
+ Please ensure the description is grammatically correct and follows the punctuation rules.
+ In the second sentence, please use `Quark` or `Quark-Engine` as the subject.
+ The third sentence could flow more smoothly. I suggest adding "Note that" at the beginning of it. Therefore, the reader knows the following sentences provide additional information.
+ Since Quark currently raises an error if the value is not specified, I think we should remove the last sentence.

What do you think?

Reply:
Makes sense Sheng, will incorporate the changes. I also suggest that not taking every word from Grammarly as a piece of advice, the sentence should be just comprehensive enough for the users.

Reply:
> BTW, updating the description usually does not make changes to the entire [build](https://github.com/quark-engine/quark-engine/tree/master/docs/build) folder. Have you manually edited any files in it?

 I belive it's because of the docs.

Reply:
@haeter525 I've made the updation.